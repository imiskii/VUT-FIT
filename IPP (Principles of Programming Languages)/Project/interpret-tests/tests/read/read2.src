.IPPcode23
DEFVAR GF@counter

READ GF@counter string

WRITE GF@counter