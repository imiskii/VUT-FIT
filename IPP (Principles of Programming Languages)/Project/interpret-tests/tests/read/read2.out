řetězec s lomítkem \ a
novým#řádkem